appinventor/components/src/com/google/appinventor/components/runtime/Notifier.java

```diff
@@ -217,8 +217,13 @@ public void ShowChooseDialog(String message, String title, final String button1T
         cancelable,
         new Runnable() {public void run() {AfterChoosing(button1Text);}},
         new Runnable() {public void run() {AfterChoosing(button2Text);}},
-        new Runnable() {public void run() {AfterChoosing(activity.getString(android.R.string.cancel));}}
-        );
+        new Runnable() {
+          public void run() {
+            AfterChoosing(activity.getString(android.R.string.cancel)); // backward compatible
```

Comment:
I'm not sure about this line. Should I keep this? @ewpatton

Reply:
Yes. I would keep this line as we would not want to break projects who relied on this functionality.